frontend/src/libraries/explorer-wamp/details.ts

```diff
@@ -32,7 +32,7 @@ export default class DetailsApi extends ExplorerApi {
           (SELECT COUNT(*) as lastDayTxCount FROM transactions
               WHERE block_timestamp > (strftime('%s','now') - 60 * 60 * 24) * 1000  ) as Daytransactions
           `
-        ]).then((it: any) => it[0]),
+        ]).then((it: any) => (it.length > 0 ? it[0] : undefined)),
```

Comment:
This is, essentially, a no-op since getting a non-existing index in JS returns undefined anyway.